src/components/Picker/index.js

```diff
@@ -21,6 +21,9 @@ const propTypes = {
     /** Error text to display */
     errorText: PropTypes.string,
 
+    /** Should the picker be styled for errors  */
+    hasError: PropTypes.bool,
+
```

Comment:
I previoulsy eliminated `hasError` prop following the [issue point 7 requirement.](https://github.com/Expensify/App/issues/7535#issue-1122420430) But now I realize that is still being used so I bring it back . What do you think?

Reply:
I'm fine with removing it. Replace existing usage  of `hasError` with `errorText`, if `errorText` is already passed just remove the `hasError`. Thanks!

cc: @luacmartins

Reply:
Yea, I actually think this should be removed everywhere like @Santhosh-Sellavel said! Sorry I wasn't clear before.

Reply:
Ok, so `Picker`'s that are still using `hasError`, e.g. [here](https://github.com/Expensify/App/blob/e11e3627d5c8f345c29f7cb470190a58df0afc54/src/pages/ReimbursementAccount/CompanyStep.js#L273), should be changed or are being changed to `errorText`?

Reply:
They should be changed

Reply:
Do you want me to change them in this PR?

Reply:
I think that should be easy enough. I see only 3 Pickers using it

Reply:
I also see `StatePicker` with `hasError`, I think they should be also changed or removed.

Reply:
Sounds good!

Reply:
Done, but I have a [concern](https://github.com/Expensify/App/pull/8765/commits/39240a99ec2c9afeb681f5d36fc2e160bbb37f15#r864303910).

Reply:
No more concerns! PR updated and ready to be reviewed.